cli/command/service/list.go

```diff
@@ -95,34 +95,51 @@ func PrintNotQuiet(out io.Writer, services []swarm.Service, nodes []swarm.Node,
 	}
 
 	running := map[string]int{}
+
+	// first key is service id, the value of first key is a map
+	// In the second map, key is node ID, value is task number of this service on this node.
+	tasksPerService := map[string]map[string]int{}
```

Comment:
Yeah, what we are interested in is number of tasks in every Service. I think we should classify tasks via different service.
If you have any better ways to implement this, please feel free to tell me.